pydarn/plotting/rtp.py

```diff
@@ -417,7 +435,7 @@ def plot_range_time(cls, dmap_data: List[dict], parameter: str = 'v',
         else:
             tick_interval = 1
         ax.xaxis.set_minor_locator(dates.MinuteLocator(interval=tick_interval))
-        if coord is Coord.SLANT_RANGE:
+        if coord is Coords.SLANT_RANGE:
             ax.yaxis.set_minor_locator(ticker.AutoMinorLocator(2))
         else:
             ax.yaxis.set_minor_locator(ticker.MultipleLocator(5))
```

Comment:
Oops sorry, I gave you the wrong line to look at! It was the minor ticks that had the issue. This should fix it, and puts one minor tick between major ticks. 

```suggestion
        if coord is Coords.SLANT_RANGE:
            ax.yaxis.set_minor_locator(ticker.AutoMinorLocator(2))
        if coord is Coords.GROUND_SCATTER_MAPPED_RANGE:
            ax.yaxis.set_minor_locator(ticker.AutoMinorLocator(2))
        else:
            ax.yaxis.set_minor_locator(ticker.MultipleLocator(5))
```